src/topology/algebra/infinite_sum.lean

```diff
@@ -73,6 +73,12 @@ by simp [has_sum, tendsto_const_nhds]
 
 lemma summable_zero : summable (λb, 0 : β → α) := has_sum_zero.summable
 
+lemma has_sum_empty [is_empty β] : has_sum f 0 :=
```

Comment:
`convert has_sum_zero` ought to be all you need here.

Can you move this lemma to be just below that lemma?